src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -14,14 +14,14 @@
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal sealed class AwaitExpressionSpiller : BoundTreeRewriterWithStackGuard
+    internal sealed class SpillSequenceSpiller : BoundTreeRewriterWithStackGuard
```

Comment:
We are likely unable to handle some expressions that are not permitted in async methods - unsafe stuff and such.

I think it is a separate workitem, but we need to make sure that all or nearly all bound expresions can be spilled.

Reply:
The switch expression can appear in an unsafe method, so spilling needs to work with almost everything. If you have specific scenarios that would illustrate the issues, that would be very helpful. Please file as a fresh issue.

---
In reply to: [178203355](https://github.com/dotnet/roslyn/pull/25697#discussion_r178203355) [](ancestors = 178203355)